src/map/packets.h

```diff
@@ -1997,4 +1997,13 @@ packet(0x96e,clif->ackmergeitems);
 	packet(0x0b6d,clif->pCashShopOpen2);
 #endif
 
+packet(0x0a52,clif->pCaptchaRegister);
```

Comment:
Packets was added in this clients

0xa52, 0xa54, 0xa5a, 0xa5c, 0xa56
2016-03-16

0xa69
2016-03-23

0xa6c
2016-03-30

Reply:
look like here no ``ifdefs`` here